ddsrecorder/src/cpp/user_interface/arguments_configuration.cpp

```diff
@@ -254,20 +255,27 @@ ProcessReturnCode parse_arguments(
                     break;
 
                 case optionIndex::ACTIVATE_DEBUG:
-                    log_filter = "DDSRECORDER";
-                    log_verbosity = eprosima::fastdds::dds::Log::Kind::Info;
+                    log_filter.set_value({
+                                {utils::VerbosityKind::Error, ""},
+                                {utils::VerbosityKind::Warning, "(DDSRECORDER|DDSPIPE)"},
+                                {utils::VerbosityKind::Info, "DDSRECORDER"}},
+                            utils::FuzzyLevelValues::fuzzy_level_fuzzy);
```

Comment:
If you are going to set the level to `fuzzy`, you don't need to write the level explicitly. `fuzzy_level_fuzzy` is the default value.